NamedPipeWrapper/NamedPipeServer.cs

```diff
@@ -74,13 +86,25 @@ public class Server<TRead, TWrite>
 		private volatile bool _shouldKeepRunning;
 		private volatile bool _isRunning;
 
+		private readonly NamedPipeServerStream _server;
+		private readonly StreamReader _reader;
+
 		/// <summary>
 		/// Constructs a new <c>NamedPipeServer</c> object that listens for client connections on the given <paramref name="pipeName"/>.
 		/// </summary>
 		/// <param name="pipeName">Name of the pipe to listen on</param>
-		public Server(string pipeName)
+		/// /// <param name="serverOnlyFlag">Flag indicating whether we are running this as server-only or not</param>
+		public Server(string pipeName, bool option = false)
```

Comment:
It's unclear what the `option` parameter is used for. What would be a better name for it? Also, it should be added to the docstring.